corehq/messaging/scheduling/tests/test_recipients.py

```diff
@@ -689,7 +689,8 @@ def _test_two_way_numbers(self, change_context_manager):
 
     @run_with_all_backends
     def test_two_way_numbers_with_signal_task(self):
-        self._test_two_way_numbers(contextlib.nullcontext())
+        nullcontext = contextlib.suppress()
```

Comment:
I didn't know about `contextlib.suppress`—always good to learn about a new Python stdlib feature! Curious why not use `contextlib.nullcontext()` instead since that seems to match this use case better? (although it does appear they are semantically equivalent when called without arguments)

Reply:
`nullcontext` is only available in python > 3.7 and we're currently on 3.6